src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { FullPost } from '../../types/FullPost';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  post: FullPost;
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by '}
+
+          {user && <UserInfo user={user} />}
+        </p>
```

Comment:
If text `posted by` rendered only for userinfo maybe would be better to move it inside `UserInfo` component ?